packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/assetModelDataStreamExplorer/assetsForAssetModelSelect/assetForAssetModelSelect.tsx

```diff
@@ -1,107 +1,43 @@
 import React from 'react';
 
-import { AssetSummary, IoTSiteWiseClient } from '@aws-sdk/client-iotsitewise';
-import Select, { SelectProps } from '@cloudscape-design/components/select';
-import { NonCancelableEventHandler } from '@cloudscape-design/components/internal/events';
-import { OptionsLoadItemsDetail } from '@cloudscape-design/components/internal/components/dropdown/interfaces';
-import { SelectedAsset } from '../useSelectedAsset';
-import {
-  isEnabled,
-  useAssetsForAssetModel,
-} from './useAssetsForAssetModel/useAssetsForAssetModel';
+import { IoTSiteWise } from '@aws-sdk/client-iotsitewise';
+import { SelectedAsset, UpdateSelectedAsset } from '../useSelectedAsset';
+import { AssetExplorer, AssetResource } from '@iot-app-kit/react-components';
+import { SelectedAssetModel } from '../useSelectedAssetModel';
 
 export type AssetForAssetModelSelectOptions = {
```

Comment:
nit: we should have comments at the top of each of these files explaining what their purpose is. A lot of these names "AssetForAssetModel" have no clear meaning